asset/css/crontab.less

```diff
@@ -0,0 +1,107 @@
+// Cron tab expression widget
+
+.crontab {
+  .ordinal {
+    display: flex;
+    flex-wrap: wrap;
+
+    .radio-label {
+      flex: 1 auto;
+    }
+
+    select {
+      flex: 1 1 auto;
+
+      &:not(select:last-of-type) {
+        margin-right: 1em;
+      }
+    }
+  }
+
+  .radio-label {
+    width: 100%;
+    margin-bottom: .5em;
+    display: flex;
+    align-items: center; // To center the radio element on safari
+  }
+
+  .number-specifier > input[type="number"] {
+    width: 5em;
+    margin: 0 1em;
+  }
+
+  .monthly, .annually, .ordinal {
+    padding: .4em;
+    border: .1em solid @crontab-list-border-color;
+    .rounded-corners(.6em);
+
+    li {
+      margin-bottom: .08em;
+    }
+  }
+
+  .list-items {
```

Comment:
I believe that `list-items` is a way too common class name. What about e.g. `.monthly > ul`?

Reply:
These rules apply not only to monthly lists, but also to weekly and yearly. However, I could prefix the class name with `crontab`, but I don't know which version is preferable!